src/store/sagas/movies.js

```diff
@@ -15,6 +20,14 @@ const getSortFilter = (filter) => {
   return sortBy;
 };
 
+const convertArrayToObject = (array) => {
```

Comment:
I think this function should be placed in something like `utils`, WDYT?

Reply:
And better to do such normalization in reducer. Just send raw data from response (into `payload`) and convert it inside reducer using your `convertArrayToObject` function (but i suggest you to rename it to `normalizeData`)